frontend/src/components/QuestionInput/ErrorAlert.tsx

```diff
@@ -1,19 +1,31 @@
+import { useEffect } from "react";
 import icons from "@newjersey/njwds/dist/img/sprite.svg";
 
-import { Alert } from "../../utils/alertUtils";
+import { Alert, ERROR_ALERT_TIMEOUT_PERIOD_IN_MS } from "../../utils/alertUtils";
 
 import styles from "./QuestionInput.module.css";
 
 interface ErrorAlertProps {
-  onClose: (id: string) => void;
+  onRemove: (id: string) => void;
   alert: Alert;
 }
 
-export const ErrorAlert = ({ onClose, alert }: ErrorAlertProps) => {
+export const ErrorAlert = ({ onRemove, alert }: ErrorAlertProps) => {
+  useEffect(() => {
+    const timeId = setTimeout(() => {
+      onRemove(alert.id);
+    }, ERROR_ALERT_TIMEOUT_PERIOD_IN_MS);
+
+    return () => {
+      clearTimeout(timeId);
+    };
+  });
```

Comment:
Pebble: Because you don't pass a dependency array, this effect will run on every render. I'm guessing your intention is to just run this once on the initial render?
```suggestion
  useEffect(() => {
    const timeId = setTimeout(() => {
      onRemove(alert.id);
    }, ERROR_ALERT_TIMEOUT_PERIOD_IN_MS);

    return () => {
      clearTimeout(timeId);
    };
  }, [alert.id, onRemove]);
```